test/suites/dev/moonbase/test-author/test-author-simple-association.ts

```diff
@@ -49,9 +49,9 @@ describeSuite({
         expect(result?.events.length === 8);
         expect(api.events.balances.Reserved.is(result?.events[1].event)).to.be.true;
         expect(api.events.authorMapping.KeysRegistered.is(result?.events[2].event)).to.be.true;
-        expect(api.events.system.NewAccount.is(result?.events[4].event)).to.be.true;
-        expect(api.events.balances.Endowed.is(result?.events[5].event)).to.be.true;
-        expect(api.events.system.ExtrinsicSuccess.is(result?.events[8].event)).to.be.true;
+        expect(api.events.system.NewAccount.is(result?.events[3].event)).to.be.true;
+        expect(api.events.balances.Endowed.is(result?.events[4].event)).to.be.true;
+        expect(api.events.system.ExtrinsicSuccess.is(result?.events[7].event)).to.be.true;
```

Comment:
Maybe we should point out in the breaking changes that an event has been removed. Not sure which one.